hs628_tools/Outliers.py

```diff
@@ -0,0 +1,66 @@
+import numpy as np
+import csv
+
+class Outliers:
+    """
+    Concept:
+    Returns list of data(list array) which fall outside of two standard deviations under the normal distribution curve.
+
+    Parameters:
+    mean, standard deviation(std)
+
+    Returns:
+    outliers
+    """
+
+    def __init__(self, mean=0, std=1, data=None): ##default values input for mean and std
+        """
+        Actions:
+        (0) Define parameters
+        (1) Load data
+        (2) Clean data by removing none numeric values("None", "Norm", and etc.)
+        (3) Convert values from string to float
+
+        Unit tests:
+          >>> "6".isnumeric()
+          True
+          >>> "None".isnumeric()
```

Comment:
The doctests you are running do not actually test your implementation, but instead they are only verifying the .isnumeric function that you use within your code. I think it will help if you actually add tests that will test your functions.

Reply:
Hi Kinjal, the unit test doc string is just to confirm the data cleaning purpose. Thanks.